CHANGELOG.md

```diff
@@ -1,3 +1,56 @@
+# v1.30.0 Release - 08/11/2021
+Note: This release comes with a new config version, `v2beta21`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Support deploy hooks for Skaffold lifecycle hooks [#6376](https://github.com/GoogleContainerTools/skaffold/pull/6376)
```

Comment:
can you link the docs? https://skaffold.dev/docs/pipeline-stages/lifecycle-hooks/

Reply:
done!!